pkg/apiclient/rollout/rollout.proto

```diff
@@ -189,6 +194,13 @@ service RolloutService {
         };
     }
 
+    rpc PromoteFullRollout(PromoteFullRolloutRequest) returns (github.com.argoproj.argo_rollouts.pkg.apis.rollouts.v1alpha1.Rollout) {
+        option (google.api.http) = {
+            put: "/api/v1/rollouts/{namespace}/{name}/promotefull"
+            body: "*"
+        };
+    }
+
```

Comment:
Instead of a new endpoint, we should just have a new `Full` option in the existing  PromoteRequest